scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ix_aern_mnk.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Area: Grand Palace of HuXzoi
+-- NPC:  Ix_aern_mnk
+-- ID: 16916815
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    local chance = GetServerVariable("[SEA]IxAern_DropRate"); -- Adjust drop rate for the items based on the organs traded.

```

Comment:
tried setting it onMobEngage? could do a check before setting it
can also have the var set in the npc's script instead of the mob's and make use of GetNPCByID to get/set the var
aaand can also add a new mobmod: MOBMOD_DROP_RATE or something

(you don't really *have* to do any of the above, i just hate server vars)